CheckArtifactConsistencyTask/task.json

```diff
@@ -27,7 +27,7 @@
     "instanceNameFormat": "Check Artifact Consistency",
     "execution":
     {
-        "Node": 
+        "Node14": 
```

Comment:
This is key for the unit tests to work. MockTestRunner uses this value to download a version of node and spawns a child process using it. This now targets node v14.